### PR TITLE
Set up

### PR DESCRIPTION
**Pull request by Nattalli**

### CODE REVIEW

**Review by romchik2612 — APPROVED** [review]

File: backend/config/settings.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+# Application definition
+
+INSTALLED_APPS = [
+    "django.contrib.admin",
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+    "django.contrib.sessions",
+    "django.contrib.messages",
+    "django.contrib.staticfiles",
+    "rest_framework",
```
Comment (antos07) [inline backend/config/settings.py:43]: Probably, it would be better to also add DRF into requirements.txt

File: README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Python 3.10+ is a must
+
+
+1. Clone the repository in the terminal:
+`git clone https://github.com/Nattalli/lteam-wine-map.git`
+2. Make the following command and populate it with required data:
+`cp .env.sample .env`
+3. Create virtual env:
+`python3.9 -m venv venv`
```
Comment (antos07) [inline README.md:13]: It should be `python3.10`, not `python3.9`

**Review by antos07 — CHANGES_REQUESTED** [review]

File: backend/config/settings.py (hunk 1 above)
Comment (Nattalli) [reply backend/config/settings.py:43]: oops, my bad

File: README.md (hunk 2 above)
Comment (Nattalli) [reply README.md:13]: agree

**Review by antos07 — APPROVED** [review]

**Pull request merged**